docs/references/nextjs/custom-signup-signin-pages.mdx

```diff
@@ -62,6 +62,28 @@ If Clerk's prebuilt components don't meet your specific needs or if you require
     ```
   </CodeBlockTabs>
 
+  ### Make the sign-up and sign-in routes public
+
+  By default, `clerkMiddleware()` makes all routes public. This step is specifically for Next.js applications that have configured `clerkMiddleware()` to make [all routes protected](/docs/references/nextjs/clerk-middleware#protect-all-routes). If you have not configured `clerkMiddleware()` to protect all routes, you can skip this step.
```

Comment:
minor nit: Any reason we're saying Next.js here? That should be implied and feels slightly redundant. This isn't enough to be a blocker though.